src/app/pages/system/support/support.component.ts

```diff
@@ -47,7 +47,18 @@ export class SupportComponent  {
       type: 'paragraph',
       name: 'support_text',
       paraText: this.sanitizer.bypassSecurityTrustHtml(
-        "Before filing a bug report or feature request, search https://redmine.ixsystems.com/projects/freenas to ensure the issue has not already been reported. If it has, add a comment to the existing issue instead of creating a new one. For enterprise-grade storage solutions and support, please visit http://www.ixsystems.com/storage/. <br><br> A Redmine account is required to file an issue. Make an account at https://redmine.ixsystems.com/account/register. Use a valid email address when registering to receive issue status updates.")
+        'Search the <a href="https://redmine.ixsystems.com/projects/freenas" \
+         target="_blank" \
+         style="text-decoration:underline;">FreeNAS issue tracker</a>
+         to ensure the issue has not already been reported before \
+         filing a bug report or feature request. If an issue has \
+         already been created, add a comment to the existing issue. \
+         Please visit the <a href="http://www.ixsystems.com/storage/" target="_blank" \
+         style="text-decoration:underline;">iXsystems storage page</a> \
+         for enterprise-grade storage solutions and support.<br><br> \
+         <a href="https://redmine.ixsystems.com/account/register" target="_blank" \
+         style="text-decoration:underline;">Create a Redmine account</a> to file an issue. Use a valid \
+         email address when registering to receive issue status updates.')
```

Comment:
There appears to be a problem with this, seems your ' quote ends improperly causing this PR to not build

Reply:
rebuild